README.md

```diff
@@ -15,9 +15,11 @@ One or all of the following:
 
 ### What are the essential skills to get into tech?
 
-- An inquisitive nature, you need to always be looking forwards
-- An insatiable appetite to learn, without learning you will stay still
+- An inquisitive nature, you need to always be looking forward.
+- An insatiable appetite to learn, without learning you will stay still.
 - The ability to simplify, Tech can be complex so you need to add simplicity (Colin Chapman once said of Motoracing cars to "Simplify and add lightness", the tech equivilent is "simplify and use less buzzwords")
+- Familiarity with the latest technologies
+- Atleast some basic knowledge of Computer Science.
```

Comment:
> Atleast some basic knowledge of Computer Science.

Thank you for the PR. I think I disagree with this one though, this can be learnt on the job